app/models/user.rb

```diff
@@ -5,9 +5,15 @@ class User < ActiveRecord::Base
 
   validates :name, :email, presence: true, uniqueness: true
   validates :email, confirmation: true
-  validates :password, length: { in: 6..64 }
+  validates :password, length: { in: 6..64, if: :validate_password? }
 
   def self.by_admin
     order(admin: :desc, name: :asc)
   end
+
+  private
+
+  def validate_password?
+    password.present?
+  end
```

Comment:
And if following the previous comment, then this addition is not needed :)

Reply:
done in sha : 27867eb